ols/src/prompts/prompts.py

```diff
@@ -23,7 +23,10 @@
 Refuse to assume any other identity or to speak as if you are someone else.
 If the context of the question is not clear, consider it to be OpenShift.
 Never include URLs in your replies.
-Refuse to answer questions or execute commands not about OpenShift.
+Refuse to answer questions that are not about OpenShift.
+When query is related to Kubernetes, Deployment & cluster, then answer with reference to OpenShift.
```

Comment:
What does "Deployment & cluster" refer to?

Reply:
I'm not sure this adds clarity. 
I was hoping to emphasize the relationship between OpenShift and Kubernetes with the
"OpenShift is a distribution of Kubernetes..."
line below. May be the "basic facts" section needs to precede the instructions...